### ISSUE TITLE
Restore-BcDatabaseFromArtifacts  failing with BcContainerHelper 4.0.10, working with 4.0.7

### ISSUE THREAD

**Issue by ichladil** [microsoft/navcontainerhelper#2866]
**Describe the issue**
Restore-BcDatabaseFromArtifacts is failing with the recent versions of BcContainerHelper 4.0.8-4.0.10. 

It seems to be working if I force older version of BcContainerHelper 4.0.7 via Import-Module bccontainerhelper -DisableNameChecking -requiredVersion 4.0.7.

**Scripts used to create container and cause the issue**
```
Import-Module bccontainerhelper -DisableNameChecking
$artifactUrl = Get-BCArtifactUrl -country cz
$databaseParams = @{    
    "databaseServer" = 'mySqlServer' 
    "databasePrefix" = 'myContainerName'
    "databaseName" = "-default"
    "multitenant" = $true
    "databaseInstance" = ''
}    

Restore-BcDatabaseFromArtifacts @databaseParams -artifactUrl $artifactUrl -async:$false 
```

**Full output of scripts**
```
BcContainerHelper version 4.0.10
BC.HelperFunctions emits usage statistics telemetry to Microsoft
Starting Database Restore job from https://bcartifacts.azureedge.net/sandbox/21.3.51409.52411/cz
C:\Program Files\WindowsPowerShell\Modules\bccontainerhelper\4.0.10\BcContainerHelper.ps1 C:\Program Files\WindowsPowerShell\Modules\bccontainerhel
per\4.0.10\Import-BcContainerHelper.ps1
Restore-BcDatabaseFromArtifacts Telemetry Correlation Id: 9a6ff80e-b6f3-42b3-ad5c-e37e24bd7ab3
The term 'C:\Program Files\WindowsPowerShell\Modules\bccontainerhelper\4.0.10\BcContainerHelper.ps1 C:\Program 
Files\WindowsPowerShell\Modules\bccontainerhelper\4.0.10\Import-BcContainerHelper.ps1' is not recognized as the name of a cmdlet, function, 
script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Program Files\WindowsPowerShell\Modules\bccontainerhelper\4.0.10\Bacpac\Restore-BcDatabaseFromArtifacts.ps1:205 char:13
+             $job | Receive-Job
+             ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Program File...ainerHelper.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
    + PSComputerName        : localhost
```

**Additional context**
- the issue can be replicated
- the script used to work for a long time
- any hints are appreciated, at the moment I have reverted the scripts to force older BcContainerHelper version 4.0.7/

**Comment by freddydk** [comment]
This is fixed in the pre-release of Containerhelper.

**Comment by freddydk** [comment]
Shipped in 5.0.0
